pkg/dns/dns_test.go

```diff
@@ -40,7 +41,7 @@ func genRandomBytes(size int) (blk []byte) {
 }
 
 func (s *DNSTestSuite) Test_DisabledServer() {
-	mockedDNSConsulClient := pbdns.NewMockDNSServiceClient(s.T())
```

Comment:
`NewMockDNSServiceClient` no longer exists in `github.com/hashicorp/consul/proto-public/pbdns`